src/Discovery/DiscoveryListView.tsx

```diff
@@ -71,6 +72,11 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
           if (props.config.features.authorization.enabled) {
             disabled = (record[props.accessibleFieldName] !== AccessLevel.ACCESSIBLE) && (record[props.accessibleFieldName] !== AccessLevel.MIXED);
           }
+          // if enableFillRequestForm is true, we allow users to check the checkbox without login
+          if (props.config.features.exportToWorkspace?.enableFillRequestForm
+            && props.config.features.exportToWorkspace.enableFillRequestForm === true) {
+            disabled = false;
```

Comment:
related to comment https://github.com/uc-cdis/data-portal/pull/1551#discussion_r1824754029

Reply:
I have changed `_medical_sample_id` to `__manifest` in [this metadata](https://qa-covid19.planx-pla.net/mds/metadata/0001-10218-buffycoat) [the](https://github.com/uc-cdis/gitops-qa/pull/3060) can you please approve it?

Reply:
see this comment https://github.com/uc-cdis/data-portal/pull/1551/files#r1832012660